client/src/components/Carousel/SliderData.js

```diff
@@ -0,0 +1,27 @@
+export const SliderData = [

```

Comment:
maybe you can fetch this data too from the backend?......cause in future changing the data would be easy instead of going through the code you can directly change it in the backend

Reply:
I didn't do it because slider data will always be limited I mean only nav links right? so why fetch from the backend.
But thanks for pointing it out I'll think of changing it.